MashUp-iOS/MashUp-iOS/Source/Screen/Setting/View/SettingMenuButton.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  SettingMenuView.swift
+//  MashUp-iOS
+//
+//  Created by 이문정 on 2022/07/15.
+//  Copyright © 2022 Mash Up Corp. All rights reserved.
+//
+
+import SnapKit
+import Then
+import UIKit
+import MashUp_Core
+import MashUp_UIKit
+
+final class SettingMenuView: BaseView {
+    private let menuTitleLabel: UILabel = UILabel()
+    private let menuTrailingImageView: UIImageView = UIImageView()
+    private let separatorView: UIView = UIView()
+    private let button: UIButton = UIButton()
+    
+    init(title: String, titleColor: UIColor) {
+        super.init(frame: .zero)
+        setupUI()
+        menuTitleLabel.textColor = titleColor
+        menuTitleLabel.text = title
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func setupUI() {
+        setupLayout()
+        setupAttribute()
+    }
+    
+    private func setupAttribute() {
+        menuTitleLabel.do {
+            $0.font = .pretendardFont(weight: .semiBold, size: 16)
+        }
+        menuTrailingImageView.do {
+            $0.backgroundColor = .red
+        }
+        separatorView.do {
+            $0.backgroundColor = .gray100
+        }
+        button.do {
+            $0.backgroundColor = .clear
+        }
+    }
+    
+    private func setupLayout() {
+        self.addSubview(menuTitleLabel)
+        self.addSubview(menuTrailingImageView)
+        self.addSubview(separatorView)
+        self.addSubview(button)
+        menuTitleLabel.snp.makeConstraints {
+            $0.leading.equalToSuperview().inset(20)
+            $0.centerY.equalToSuperview()
+        }
+        menuTrailingImageView.snp.makeConstraints {
+            $0.trailing.equalToSuperview().inset(20)
+            $0.height.width.equalTo(20)
+            $0.centerY.equalTo(menuTitleLabel.snp.centerY)
```

Comment:
```suggestion
            $0.centerY.equalTo(menuTitleLabel)
```
이렇게 축약할 수도 있습니답 참고만 해주세요 🙇‍♂️